docs/5.0/preview/database-access.md

```diff
@@ -149,34 +208,70 @@ ssh_service:
 Start the service:
 
 ```sh
-$ teleport start --debug --config=/path/to/teleport.yaml
+$ teleport start -d --config=/path/to/teleport.yaml
+```
+
+### Start Database Service with CLI Flags
+
+For a quick try-out, Teleport database service doesn't require a configuration
+file and can be launched using a single CLI command:
+
+```sh
+$ teleport start -d \
+   --roles=database \
+   --token=cbdeeab9-6f88-436d-a673-44d14bd86bb7 \
+   --auth-server=teleport.example.com:3080 \
+   --db-name=test \
+   --db-protocol=postgres \
+   --db-uri=db.example.com:5432 \
+   --labels=env=test
+```
+
+Note that the `--auth-server` flag must point to cluster's proxy endpoint
+because database service always connects back to the cluster over reverse
+tunnel.
+
+Instead of using a static auth token, a short-lived dynamic token can also
+be generated for a database service:
+
+```sh
+$ tctl tokens add \
+    --type=database \
+    --db-name=test \
+    --db-protocol=postgres \
+    --db-uri=db.example.com:5432
 ```
 
-### Start Teleport Database Service
+### Start Database Service with Config File
 
-Create configuration file for a database service:
+Below is an example of a database service configuration file that proxies
+a single AWS Aurora database:
 
 ```yaml
 teleport:
   data_dir: /var/lib/teleport-db
   nodename: test
   # Auth token to connect to the auth server.
-  auth_token: qwe123
-  # Proxy address to connect to.
+  auth_token: cbdeeab9-6f88-436d-a673-44d14bd86bb7
+  # Proxy address to connect to. Note that it has to be the proxy address
+  # because database service always connects to the cluster over reverse
+  # tunnel framework.
   auth_servers:
   - teleport.example.com:3080
 db_service:
   enabled: "yes"
+  # This section contains definitions of all databases proxied by this
+  # service, can contain multiple items.
   databases:
     # Name of the database proxy instance, used to reference in CLI.
-  - name: "postgres-aurora"
+  - name: "aurora"
     # Free-form description of the database proxy instance.
     description: "AWS Aurora instance of PostgreSQL 13.0"
     # Database protocol.
     protocol: "postgres"
-    # Database address.
+    # Database address, example of a AWS Aurora endpoint in this case.
     uri: "postgres-aurora-instance-1.xxx.us-east-1.rds.amazonaws.com:5432"
-    # AWS specific configuration.
+    # AWS specific configuration, only required for RDS and Aurora.
     aws:
       # Region the database is deployed in.
       region: us-east-1
```

Comment:
Do we really need this extra setting if we already know the location of the database via the uri? 

postgres-aurora-instance-1.xxx.**us-east-1**.rds.amazonaws.com:5432

Reply:
I thought about this. The thing is, I'm also using this field to tell that this is actually RDS/Aurora instance (to use IAM auth). I guess we could also just try to determine this from the URI as well, but at that point I feel like it would be trying to be too smart and deduce too much which may lead to confusing behaviors (i.e. could you have a different URI for RDS, like a cname? I don't know)... so I usually prefer to be explicit. It is pretty tempting though to get rid of an extra setting.